// @clerk/localizations/dist/es-ES.js
"use strict";
var __defProp = Object.defineProperty;
var __getOwnPropDesc = Object.getOwnPropertyDescriptor;
var __getOwnPropNames = Object.getOwnPropertyNames;
var __hasOwnProp = Object.prototype.hasOwnProperty;
var __export = (target, all) => {
  for (var name in all)
    __defProp(target, name, { get: all[name], enumerable: true });
};
var __copyProps = (to, from, except, desc) => {
  if (from && typeof from === "object" || typeof from === "function") {
    for (let key of __getOwnPropNames(from))
      if (!__hasOwnProp.call(to, key) && key !== except)
        __defProp(to, key, { get: () => from[key], enumerable: !(desc = __getOwnPropDesc(from, key)) || desc.enumerable });
  }
  return to;
};
var __toCommonJS = (mod) => __copyProps(__defProp({}, "__esModule", { value: true }), mod);

// src/es-ES.ts
var es_ES_exports = {};
__export(es_ES_exports, {
  esES: () => esES
});
module.exports = __toCommonJS(es_ES_exports);
var esES = {
  locale: "es-ES",
  __experimental_formFieldLabel__passkeyName: void 0,
  backButton: "Atr\xE1s",
  badge__default: "Por defecto",
  badge__otherImpersonatorDevice: "Otro dispositivo de imitaci\xF3n",
  badge__primary: "Primario",
  badge__requiresAction: "Requiere acci\xF3n",
  badge__thisDevice: "Este dispositivo",
  badge__unverified: "No confirmado",
  badge__userDevice: "Dispositivo de usuario",
  badge__you: "Usted",
  createOrganization: {
    formButtonSubmit: "Crear organizaci\xF3n",
    invitePage: {
      formButtonReset: "Saltar"
    },
    title: "Crear organizaci\xF3n"
  },
  dates: {
    lastDay: "Ayer a las {{ date | timeString('es-ES') }}",
    next6Days: "{{ date | weekday('es-ES','long') }} a las {{ date | timeString('es-ES') }}",
    nextDay: "Ma\xF1ana a las {{ date | timeString('es-ES') }}",
    numeric: "{{ date | numeric('es-ES') }}",
    previous6Days: "\xDAltimo {{ date | weekday('es-ES','long') }} en {{ date | timeString('es-ES') }}",
    sameDay: "Hoy a las {{ date | timeString('es-ES') }}"
  },
  dividerText: "o",
  footerActionLink__useAnotherMethod: "Usar otro m\xE9todo",
  footerPageLink__help: "Ayuda",
  footerPageLink__privacy: "Privacidad",
  footerPageLink__terms: "T\xE9rminos",
  formButtonPrimary: "Continuar",
  formButtonPrimary__verify: "Verificar",
  formFieldAction__forgotPassword: "Has olvidado tu contrase\xF1a?",
  formFieldError__matchingPasswords: "Las contrase\xF1as coinciden.",
  formFieldError__notMatchingPasswords: "Las contrase\xF1as no coinciden.",
  formFieldError__verificationLinkExpired: "El enlace de verificaci\xF3n ha expirado. Por favor solicite uno nuevo.",
  formFieldHintText__optional: "Opcional",
  formFieldHintText__slug: "Un slug es un ID legible que debe ser \xFAnico. Es com\xFAnmente usado en URLs.",
  formFieldInputPlaceholder__backupCode: "",
  formFieldInputPlaceholder__confirmDeletionUserAccount: "Eliminar cuenta",
  formFieldInputPlaceholder__emailAddress: "",
  formFieldInputPlaceholder__emailAddress_username: "",
  formFieldInputPlaceholder__emailAddresses: "Ingrese o pegue una o m\xE1s direcciones de correo electr\xF3nico, separadas por espacios o comas",
  formFieldInputPlaceholder__firstName: "",
  formFieldInputPlaceholder__lastName: "",
  formFieldInputPlaceholder__organizationDomain: "",
  formFieldInputPlaceholder__organizationDomainEmailAddress: "",
  formFieldInputPlaceholder__organizationName: "",
  formFieldInputPlaceholder__organizationSlug: "",
  formFieldInputPlaceholder__password: "",
  formFieldInputPlaceholder__phoneNumber: "",
  formFieldInputPlaceholder__username: "",
  formFieldLabel__automaticInvitations: "Activar invitaciones autom\xE1ticas para este dominio",
  formFieldLabel__backupCode: "C\xF3digo de respaldo",
  formFieldLabel__confirmDeletion: "Confirmaci\xF3n",
  formFieldLabel__confirmPassword: "Confirme la contrase\xF1a",
  formFieldLabel__currentPassword: "Contrase\xF1a actual",
  formFieldLabel__emailAddress: "Correo electr\xF3nico",
  formFieldLabel__emailAddress_username: "Correo electr\xF3nico o nombre de usuario",
  formFieldLabel__emailAddresses: "Direcciones de correo",
  formFieldLabel__firstName: "Nombre",
  formFieldLabel__lastName: "Apellidos",
  formFieldLabel__newPassword: "Nueva contrase\xF1a",
  formFieldLabel__organizationDomain: "Dominio",
  formFieldLabel__organizationDomainDeletePending: "Borrar invitaciones y sugerencias pendientes",
  formFieldLabel__organizationDomainEmailAddress: "Correo de verificaci\xF3n",
  formFieldLabel__organizationDomainEmailAddressDescription: "Ingrese una direcci\xF3n de correo electr\xF3nico bajo este dominio para recibir un c\xF3digo y verificarlo.",
  formFieldLabel__organizationName: "Nombre de la Organizaci\xF3n",
  formFieldLabel__organizationSlug: "Slug",
  formFieldLabel__passkeyName: void 0,
  formFieldLabel__password: "Contrase\xF1a",
  formFieldLabel__phoneNumber: "N\xFAmero telef\xF3nico",
  formFieldLabel__role: "Rol",
  formFieldLabel__signOutOfOtherSessions: "Cerrar sesi\xF3n en todos los dem\xE1s dispositivos",
  formFieldLabel__username: "Nombre de usuario",
  impersonationFab: {
    action__signOut: "Cerrar",
    title: "Registrado como {{identifier}}"
  },
  membershipRole__admin: "Administrador",
  membershipRole__basicMember: "Miembro",
  membershipRole__guestMember: "Invitado",
  organizationList: {
    action__createOrganization: "Crear organizaci\xF3n",
    action__invitationAccept: "Join",
    action__suggestionsAccept: "Solicitud a unirse",
    createOrganization: "Crear Organizaci\xF3n",
    invitationAcceptedLabel: "Unido",
    subtitle: "to continue to {{applicationName}}",
    suggestionsAcceptedLabel: "Aprobaci\xF3n pendiente",
    title: "Choose an account",
    titleWithoutPersonal: "Escoja una organizaci\xF3n"
  },
  organizationProfile: {
    badge__automaticInvitation: "Invitaciones autom\xE1ticas",
    badge__automaticSuggestion: "Sugerencias autom\xE1ticas",
    badge__manualInvitation: "Sin inscripci\xF3n autom\xE1tica",
    badge__unverified: "No verificado",
    createDomainPage: {
      subtitle: "Agregue el dominio para verificar. Los usuarios con direcciones de correo electr\xF3nico en este dominio pueden unirse a la organizaci\xF3n autom\xE1ticamente o solicitar unirse.",
      title: "Agregar dominio"
    },
    invitePage: {
      detailsTitle__inviteFailed: "No se pudieron enviar las invitaciones. Solucione lo siguiente y vuelva a intentarlo:",
      formButtonPrimary__continue: "Enviar invitaciones",
      selectDropdown__role: "Seleccionar rol",
      subtitle: "Invitar nuevos miembros a esta organizaci\xF3n",
      successMessage: "Invitaciones enviadas con \xE9xito",
      title: "Invitar miembros"
    },
    membersPage: {
      action__invite: "Invitar",
      activeMembersTab: {
        menuAction__remove: "Quitar miembro",
        tableHeader__actions: "",
        tableHeader__joined: "Unido",
        tableHeader__role: "Rol",
        tableHeader__user: "Usuario"
      },
      detailsTitle__emptyRow: "No hay miembros para mostrar",
      invitationsTab: {
        autoInvitations: {
          headerSubtitle: "Invite a usuarios conectando un dominio de correo electr\xF3nico con su organizaci\xF3n. Cualquiera que se registre con un dominio de correo electr\xF3nico coincidente podr\xE1 unirse a la organizaci\xF3n en cualquier momento.",
          headerTitle: "Invitaciones autom\xE1ticas",
          primaryButton: "Gestionar dominios verificados"
        },
        table__emptyRow: "Sin invitaciones para mostrar"
      },
      invitedMembersTab: {
        menuAction__revoke: "Revocar invitaci\xF3n",
        tableHeader__invited: "Invitado"
      },
      requestsTab: {
        autoSuggestions: {
          headerSubtitle: "Los usuarios que se registren con un dominio de correo electr\xF3nico coincidente podr\xE1n ver una sugerencia para solicitar unirse a su organizaci\xF3n.",
          headerTitle: "Sugerencias autom\xE1ticas",
          primaryButton: "Gestionar dominios verificados"
        },
        menuAction__approve: "Aprobar",
        menuAction__reject: "Rechazar",
        tableHeader__requested: "Acceso solicitado",
        table__emptyRow: "Sin solicitudes para mostrar"
      },
      start: {
        headerTitle__invitations: "Invitaciones",
        headerTitle__members: "Miembros",
        headerTitle__requests: "Solicitudes"
      }
    },
    navbar: {
      description: "Gestione su organizaci\xF3n.",
      general: "General",
      members: "Miembros",
      title: "Organizaci\xF3n"
    },
    profilePage: {
      dangerSection: {
        deleteOrganization: {
          actionDescription: 'Type "{{organizationName}}" below to continue.',
          messageLine1: "Are you sure you want to delete this organization?",
          messageLine2: "This action is permanent and irreversible.",
          successMessage: "You have deleted the organization.",
          title: "Delete organization"
        },
        leaveOrganization: {
          actionDescription: 'Type "{{organizationName}}" below to continue.',
          messageLine1: "\xBFEst\xE1 seguro de que desea abandonar esta organizaci\xF3n? Perder\xE1 el acceso a esta organizaci\xF3n y sus aplicaciones.",
          messageLine2: "Esta acci\xF3n es permanente e irreversible.",
          successMessage: "Has dejado la organizaci\xF3n.",
          title: "Abandonar la organizaci\xF3n"
        },
        title: "Peligro"
      },
      domainSection: {
        menuAction__manage: "Manage",
        menuAction__remove: "Delete",
        menuAction__verify: "Verify",
        primaryButton: "Add domain",
        subtitle: "Allow users to join the organization automatically or request to join based on a verified email domain.",
        title: "Verified domains"
      },
      successMessage: "La organizaci\xF3n ha sido actualizada.",
      title: "Perfil de la organizaci\xF3n"
    },
    removeDomainPage: {
      messageLine1: "The email domain {{domain}} will be removed.",
      messageLine2: "Users won\u2019t be able to join the organization automatically after this.",
      successMessage: "{{domain}} has been removed.",
      title: "Remove domain"
    },
    start: {
      headerTitle__general: "General",
      headerTitle__members: "Miembros",
      profileSection: {
        primaryButton: "",
        title: "Organization Profile",
        uploadAction__title: "Logo"
      }
    },
    verifiedDomainPage: {
      dangerTab: {
        calloutInfoLabel: "Removing this domain will affect invited users.",
        removeDomainActionLabel__remove: "Remove domain",
        removeDomainSubtitle: "Remove this domain from your verified domains",
        removeDomainTitle: "Remove domain"
      },
      enrollmentTab: {
        automaticInvitationOption__description: "Users are automatically invited to join the organization when they sign-up and can join anytime.",
        automaticInvitationOption__label: "Automatic invitations",
        automaticSuggestionOption__description: "Users receive a suggestion to request to join, but must be approved by an admin before they are able to join the organization.",
        automaticSuggestionOption__label: "Automatic suggestions",
        calloutInfoLabel: "Changing the enrollment mode will only affect new users.",
        calloutInvitationCountLabel: "Pending invitations sent to users: {{count}}",
        calloutSuggestionCountLabel: "Pending suggestions sent to users: {{count}}",
        manualInvitationOption__description: "Users can only be invited manually to the organization.",
        manualInvitationOption__label: "No automatic enrollment",
        subtitle: "Choose how users from this domain can join the organization."
      },
      start: {
        headerTitle__danger: "Danger",
        headerTitle__enrollment: "Enrollment options"
      },
      subtitle: "The domain {{domain}} is now verified. Continue by selecting enrollment mode.",
      title: "Update {{domain}}"
    },
    verifyDomainPage: {
      formSubtitle: "Enter the verification code sent to your email address",
      formTitle: "Verification code",
      resendButton: "Didn't receive a code? Resend",
      subtitle: "The domain {{domainName}} needs to be verified via email.",
      subtitleVerificationCodeScreen: "A verification code was sent to {{emailAddress}}. Enter the code to continue.",
      title: "Verify domain"
    }
  },
  organizationSwitcher: {
    action__createOrganization: "Crear Organizaci\xF3n",
    action__invitationAccept: "Join",
    action__manageOrganization: "Administrar Organizaci\xF3n",
    action__suggestionsAccept: "Request to join",
    notSelected: "Ninguna organizaci\xF3n seleccionada",
    personalWorkspace: "Espacio de trabajo personal",
    suggestionsAcceptedLabel: "Pending approval"
  },
  paginationButton__next: "Pr\xF3ximo",
  paginationButton__previous: "Previo",
  paginationRowText__displaying: "Mostrando",
  paginationRowText__of: "de",
  signIn: {
    accountSwitcher: {
      action__addAccount: "Add account",
      action__signOutAll: "Sign out of all accounts",
      subtitle: "Select the account with which you wish to continue.",
      title: "Choose an account"
    },
    alternativeMethods: {
      actionLink: "Consigue ayuda",
      actionText: "Don\u2019t have any of these?",
      blockButton__backupCode: "Usa un c\xF3digo de respaldo",
      blockButton__emailCode: "Enviar c\xF3digo a{{identifier}}",
      blockButton__emailLink: "Enviar enlace a{{identifier}}",
      blockButton__passkey: void 0,
      blockButton__password: "Entra con tu contrase\xF1a",
      blockButton__phoneCode: "Enviar c\xF3digo a{{identifier}}",
      blockButton__totp: "Usa tu aplicaci\xF3n de autenticaci\xF3n",
      getHelp: {
        blockButton__emailSupport: "Soporte de correo electr\xF3nico",
        content: "Si tiene dificultades para iniciar sesi\xF3n en su cuenta, env\xEDenos un correo electr\xF3nico y trabajaremos con usted para restablecer el acceso lo antes posible.",
        title: "Consigue ayuda"
      },
      subtitle: "Facing issues? You can use any of these methods to sign in.",
      title: "Usa otro m\xE9todo"
    },
    backupCodeMfa: {
      subtitle: "para continuar a {{applicationName}}",
      title: "Introduce un c\xF3digo de seguridad"
    },
    emailCode: {
      formTitle: "C\xF3digo de verificaci\xF3n",
      resendButton: "Re-enviar c\xF3digo",
      subtitle: "para continuar a {{applicationName}}",
      title: "Revise su correo electr\xF3nico"
    },
    emailLink: {
      expired: {
        subtitle: "Regrese a la pesta\xF1a original para continuar.",
        title: "Este enlace de verificaci\xF3n ha expirado"
      },
      failed: {
        subtitle: "Regrese a la pesta\xF1a original para continuar.",
        title: "Este enlace de verificaci\xF3n es invalido"
      },
      formSubtitle: "Utilice el enlace de verificaci\xF3n enviado a su correo electr\xF3nico",
      formTitle: "Enlace de verificaci\xF3n",
      loading: {
        subtitle: "Ser\xE1s redirigido pronto",
        title: "Iniciando sesi\xF3n..."
      },
      resendButton: "Reenviar enlace",
      subtitle: "para continuar a {{applicationName}}",
      title: "Revise su correo electr\xF3nico",
      unusedTab: {
        title: "Puede cerrar esta pesta\xF1a"
      },
      verified: {
        subtitle: "Ser\xE1s redirigido pronto",
        title: "Inici\xF3 sesi\xF3n con \xE9xito"
      },
      verifiedSwitchTab: {
        subtitle: "Regrese a la pesta\xF1a original para continuar",
        subtitleNewTab: "Regrese a la pesta\xF1a reci\xE9n abierta para continuar",
        titleNewTab: "Inici\xF3 sesi\xF3n en otra pesta\xF1a"
      }
    },
    forgotPassword: {
      formTitle: "Reset password code",
      resendButton: "Didn't receive a code? Resend",
      subtitle: "to reset your password",
      subtitle_email: "First, enter the code sent to your email ID",
      subtitle_phone: "First, enter the code sent to your phone",
      title: "Reset password"
    },
    forgotPasswordAlternativeMethods: {
      blockButton__resetPassword: "Reset your password",
      label__alternativeMethods: "Or, sign in with another method",
      title: "Forgot Password?"
    },
    noAvailableMethods: {
      message: "No se puede continuar con el inicio de sesi\xF3n. No hay ning\xFAn factor de autenticaci\xF3n disponible.",
      subtitle: "Ocurri\xF3 un error",
      title: "No puedo iniciar sesi\xF3n"
    },
    passkey: {
      subtitle: void 0,
      title: void 0
    },
    password: {
      actionLink: "Use otro m\xE9todo",
      subtitle: "para continuar a {{applicationName}}",
      title: "Introduzca su contrase\xF1a"
    },
    passwordPwned: {
      title: void 0
    },
    phoneCode: {
      formTitle: "C\xF3digo de verificaci\xF3n",
      resendButton: "Reenviar c\xF3digo",
      subtitle: "para continuar a {{applicationName}}",
      title: "Revisa tu tel\xE9fono"
    },
    phoneCodeMfa: {
      formTitle: "C\xF3digo de verificaci\xF3n",
      resendButton: "Reenviar c\xF3digo",
      subtitle: "",
      title: "Revisa tu tel\xE9fono"
    },
    resetPassword: {
      formButtonPrimary: "Reset Password",
      requiredMessage: "For security reasons, it is required to reset your password.",
      successMessage: "Your password was successfully changed. Signing you in, please wait a moment.",
      title: "Set new password"
    },
    resetPasswordMfa: {
      detailsLabel: "We need to verify your identity before resetting your password."
    },
    start: {
      actionLink: "Reg\xEDstrese",
      actionLink__use_email: "Use email",
      actionLink__use_email_username: "Use email or username",
      actionLink__use_passkey: void 0,
      actionLink__use_phone: "Use phone",
      actionLink__use_username: "Use username",
      actionText: "\xBFNo tiene cuenta?",
      subtitle: "para continuar a {{applicationName}}",
      title: "Entrar"
    },
    totpMfa: {
      formTitle: "C\xF3digo de verificaci\xF3n",
      subtitle: "",
      title: "Verificaci\xF3n de dos pasos"
    }
  },
  signInEnterPasswordTitle: "Ingresa tu contrase\xF1a",
  signUp: {
    continue: {
      actionLink: "Entrar",
      actionText: "\xBFTiene una cuenta?",
      subtitle: "para continuar a {{applicationName}}",
      title: "Rellene los campos que faltan"
    },
    emailCode: {
      formSubtitle: "Introduzca el c\xF3digo de verificaci\xF3n enviado a su correo electr\xF3nico",
      formTitle: "C\xF3digo de verificaci\xF3n",
      resendButton: "Re-enviar c\xF3digo",
      subtitle: "para continuar a {{applicationName}}",
      title: "Verifique su correo electr\xF3nico"
    },
    emailLink: {
      formSubtitle: "Utilice el enlace de verificaci\xF3n enviado a su direcci\xF3n de correo electr\xF3nico",
      formTitle: "Enlace de verificaci\xF3n",
      loading: {
        title: "Registrando..."
      },
      resendButton: "Reenviar enlace",
      subtitle: "para continuar a {{applicationName}}",
      title: "Verifica tu correo electr\xF3nico",
      verified: {
        title: "Registrado con \xE9xito"
      },
      verifiedSwitchTab: {
        subtitle: "Regrese a la pesta\xF1a reci\xE9n abierta para continuar",
        subtitleNewTab: "Volver a la pesta\xF1a anterior para continuar",
        title: "Correo electr\xF3nico verificado con \xE9xito"
      }
    },
    phoneCode: {
      formSubtitle: "Introduzca el c\xF3digo de verificaci\xF3n enviado a su n\xFAmero de tel\xE9fono",
      formTitle: "C\xF3digo de verificaci\xF3n",
      resendButton: "Re-enviar c\xF3digo",
      subtitle: "para continuar a {{applicationName}}",
      title: "Verifique su tel\xE9fono"
    },
    start: {
      actionLink: "Entrar",
      actionText: "\xBFTienes una cuenta?",
      subtitle: "para continuar a {{applicationName}}",
      title: "Crea tu cuenta"
    }
  },
  socialButtonsBlockButton: "Continuar con {{provider|titleize}}",
  unstable__errors: {
    captcha_invalid: "Sign up unsuccessful due to failed security validations. Please refresh the page to try again or reach out to support for more assistance.",
    captcha_unavailable: "Sign up unsuccessful due to failed bot validation. Please refresh the page to try again or reach out to support for more assistance.",
    form_code_incorrect: "",
    form_identifier_exists: "",
    form_identifier_exists__email_address: void 0,
    form_identifier_exists__phone_number: void 0,
    form_identifier_exists__username: void 0,
    form_identifier_not_found: "",
    form_param_format_invalid: "",
    form_param_format_invalid__email_address: "Email address must be a valid email address.",
    form_param_format_invalid__phone_number: "Phone number must be in a valid international format",
    form_param_max_length_exceeded__first_name: "First name should not exceed 256 characters.",
    form_param_max_length_exceeded__last_name: "Last name should not exceed 256 characters.",
    form_param_max_length_exceeded__name: "Name should not exceed 256 characters.",
    form_param_nil: "",
    form_password_incorrect: "",
    form_password_length_too_short: "",
    form_password_not_strong_enough: "Your password is not strong enough.",
    form_password_pwned: "",
    form_password_pwned__sign_in: void 0,
    form_password_size_in_bytes_exceeded: "Your password has exceeded the maximum number of bytes allowed, please shorten it or remove some special characters.",
    form_password_validation_failed: "Incorrect Password",
    form_username_invalid_character: "",
    form_username_invalid_length: "",
    identification_deletion_failed: "You cannot delete your last identification.",
    not_allowed_access: "",
    passkey_already_exists: void 0,
    passkey_not_supported: void 0,
    passkey_registration_cancelled: void 0,
    passkey_retrieval_cancelled: void 0,
    passkey_pa_not_supported: void 0,
    passwordComplexity: {
      maximumLength: "",
      minimumLength: "",
      requireLowercase: "",
      requireNumbers: "",
      requireSpecialCharacter: "",
      requireUppercase: "",
      sentencePrefix: ""
    },
    phone_number_exists: "This phone number is taken. Please try another.",
    zxcvbn: {
      couldBeStronger: "Your password works, but could be stronger. Try adding more characters.",
      goodPassword: "Your password meets all the necessary requirements.",
      notEnough: "Tu contrase\xF1a no es lo suficientemente segura.",
      suggestions: {
        allUppercase: "Escribe en may\xFAsculas algunas, pero no todas las letras.",
        anotherWord: "A\xF1ade m\xE1s palabras que sean menos comunes.",
        associatedYears: "Evita a\xF1os asociados contigo.",
        capitalization: "Escribe en may\xFAsculas alguna letra m\xE1s adem\xE1s de la primera.",
        dates: "Evita fechas que est\xE9n asociadas contigo.",
        l33t: "Evita sustituciones predecibles como '@' por 'a'",
        longerKeyboardPattern: "Usa patrones de teclado m\xE1s largos y cambia la direcci\xF3n de escritura varias veces.",
        noNeed: "Puedes crear contrase\xF1as seguras sin usar s\xEDmbolos, n\xFAmeros o may\xFAsculas.",
        pwned: "Si utiliza esta contrase\xF1a en otro lugar, deber\xEDa cambiarla.",
        recentYears: "Evita a\xF1os recientes.",
        repeated: "Evitas palabras y letras repetidas.",
        reverseWords: "Evita palabras comunes escritas al rev\xE9s",
        sequences: "Evita secuencias de letras comunes.",
        useWords: "Usa varias palabras, pero evita frases comunes."
      },
      warnings: {
        common: "Es una contrase\xF1a usada com\xFAnmente.",
        commonNames: "Nombre y apellidos comunes son f\xE1ciles de adivinar.",
        dates: "Las fechas son f\xE1ciles de adivinar.",
        extendedRepeat: 'Patrones repetidos como "abcabcabc" son f\xE1ciles de adivinar.',
        keyPattern: "Patrones cortos son f\xE1ciles de adivinar.",
        namesByThemselves: "Nombres o apellidos a solas son f\xE1ciles de adivinar.",
        pwned: "Su contrase\xF1a fue expuesta por una violaci\xF3n de datos en Internet.",
        recentYears: "Los a\xF1os recientes son f\xE1ciles de adivinar.",
        sequences: 'Patrones comunes como "abc" son f\xE1ciles de adivinar',
        similarToCommon: "Es similar a una contrase\xF1a usada habitualmente.",
        simpleRepeat: 'Caracteres repetidos como "aaa" son f\xE1ciles de adivinar',
        straightRow: "Teclas consecutivas en tu teclado son f\xE1ciles de adivinar.",
        topHundred: "Es una contrase\xF1a usada con mucha frecuencia.",
        topTen: "Es de las contrase\xF1as m\xE1s usadas.",
        userInputs: "No deber\xEDa haber datos personales o relacionados con esta p\xE1gina.",
        wordByItself: "Palabras \xFAnicas son f\xE1ciles de adivinar."
      }
    }
  },
  userButton: {
    action__addAccount: "A\xF1adir cuenta",
    action__manageAccount: "Administrar cuenta",
    action__signOut: "Cerrar sesi\xF3n",
    action__signOutAll: "Salir de todas las cuentas"
  },
  userProfile: {
    __experimental_passkeyScreen: {
      removeResource: {
        messageLine1: void 0,
        title: void 0
      },
      subtitle__rename: void 0,
      title__rename: void 0
    },
    backupCodePage: {
      actionLabel__copied: "Copiado!",
      actionLabel__copy: "Copiar todo",
      actionLabel__download: "Descargar .txt",
      actionLabel__print: "Imprimir",
      infoText1: "Se habilitar\xE1n c\xF3digos de respaldo para esta cuenta.",
      infoText2: "Mantenga los c\xF3digos de respaldo en secreto y gu\xE1rdelos de forma segura. Puede regenerar c\xF3digos de respaldo si sospecha que se han visto comprometidos.",
      subtitle__codelist: "Guardelos de forma segura y mant\xE9ngalos en secreto.",
      successMessage: "Los c\xF3digos de respaldo ahora est\xE1n habilitados. Puede usar uno de estos para iniciar sesi\xF3n en su cuenta, si pierde el acceso a su dispositivo de autenticaci\xF3n. Cada c\xF3digo solo se puede utilizar una vez.",
      successSubtitle: "Puede usar uno de estos para iniciar sesi\xF3n en su cuenta, si pierde el acceso a su dispositivo de autenticaci\xF3n.",
      title: "Agregar verificaci\xF3n de c\xF3digo de respaldo",
      title__codelist: "C\xF3digos de respaldo"
    },
    connectedAccountPage: {
      formHint: "Seleccione un proveedor para conectar su cuenta.",
      formHint__noAccounts: "No hay proveedores de cuentas externas disponibles.",
      removeResource: {
        messageLine1: "{{identifier}} ser\xE1 eliminado de esta cuenta.",
        messageLine2: "Ya no podr\xE1 usar esta cuenta activa y las funciones dependientes ya no funcionar\xE1n.",
        successMessage: "{{connectedAccount}} ha sido eliminado de su cuenta.",
        title: "Eliminar cuenta conectada"
      },
      socialButtonsBlockButton: "Conectar cuenta de {{provider|titleize}}",
      successMessage: "El proveedor ha sido agregado a su cuenta",
      title: "Agregar cuenta conectada"
    },
    deletePage: {
      actionDescription: 'Escribe "Eliminar cuenta" a continuaci\xF3n para continuar.',
      confirm: "Eliminar cuenta",
      messageLine1: "\xBFEst\xE1s seguro de que quieres eliminar tu cuenta?",
      messageLine2: "Esta acci\xF3n es permanente e irreversible.",
      title: "Eliminar cuenta"
    },
    emailAddressPage: {
      emailCode: {
        formHint: "A esta direcci\xF3n de correo electr\xF3nico se le enviar\xE1 un correo electr\xF3nico con un C\xF3digo de verificaci\xF3n.",
        formSubtitle: "Introduzca el c\xF3digo de verificaci\xF3n enviado a {{identifier}}",
        formTitle: "C\xF3digo de verificaci\xF3n",
        resendButton: "Re-enviar c\xF3digo",
        successMessage: "El correo electr\xF3nico {{identifier}} se ha agregado a su cuenta."
      },
      emailLink: {
        formHint: "Se enviar\xE1 un correo electr\xF3nico con un enlace de verificaci\xF3n a esta direcci\xF3n de correo electr\xF3nico.",
        formSubtitle: "Haga clic en el enlace de verificaci\xF3n en el correo electr\xF3nico enviado a {{identifier}}",
        formTitle: "Enlace de verificaci\xF3n",
        resendButton: "Reenviar enlace",
        successMessage: "El correo electr\xF3nico {{identifier}} se ha agregado a su cuenta."
      },
      removeResource: {
        messageLine1: "{{identifier}} ser\xE1 eliminado de esta cuenta.",
        messageLine2: "Ya no podr\xE1 iniciar sesi\xF3n con esta direcci\xF3n de correo electr\xF3nico.",
        successMessage: "{{emailAddress}} ha sido eliminado de su cuenta.",
        title: "Eliminar direcci\xF3n de correo electr\xF3nico"
      },
      title: "Agregar direcci\xF3n de correo electr\xF3nico",
      verifyTitle: "Verify email address"
    },
    formButtonPrimary__add: "Add",
    formButtonPrimary__continue: "Continuar",
    formButtonPrimary__finish: "Terminar",
    formButtonPrimary__remove: "Remove",
    formButtonPrimary__save: "Save",
    formButtonReset: "Cancelar",
    mfaPage: {
      formHint: "Seleccione un m\xE9todo para agregar.",
      title: "Agregar verificaci\xF3n en dos pasos"
    },
    mfaPhoneCodePage: {
      backButton: "Use existing number",
      primaryButton__addPhoneNumber: "Agregar un n\xFAmero de tel\xE9fono",
      removeResource: {
        messageLine1: "{{identifier}} dejar\xE1 de recibir el C\xF3digo de verificaci\xF3n al iniciar sesi\xF3n.",
        messageLine2: "Es posible que su cuenta no sea tan segura. \xBFEst\xE1s seguro de que quieres continuar?",
        successMessage: "Se elimin\xF3 la verificaci\xF3n de dos pasos del c\xF3digo SMS para {{mfaPhoneCode}}",
        title: "Eliminar la verificaci\xF3n en dos pasos"
      },
      subtitle__availablePhoneNumbers: "Seleccione un n\xFAmero de tel\xE9fono para registrarse para la verificaci\xF3n en dos pasos del c\xF3digo SMS.",
      subtitle__unavailablePhoneNumbers: "No hay n\xFAmeros de tel\xE9fono disponibles para registrarse para la verificaci\xF3n en dos pasos del c\xF3digo SMS.",
      successMessage1: "When signing in, you will need to enter a verification code sent to this phone number as an additional step.",
      successMessage2: "Save these backup codes and store them somewhere safe. If you lose access to your authentication device, you can use backup codes to sign in.",
      successTitle: "SMS code verification enabled",
      title: "Agregar verificaci\xF3n de c\xF3digo SMS"
    },
    mfaTOTPPage: {
      authenticatorApp: {
        buttonAbleToScan__nonPrimary: "Escanea el c\xF3digo QR en su lugar",
        buttonUnableToScan__nonPrimary: "\xBFNo puedes escanear el c\xF3digo QR?",
        infoText__ableToScan: "Configure un nuevo m\xE9todo de inicio de sesi\xF3n en su aplicaci\xF3n de autenticaci\xF3n y escanee el siguiente c\xF3digo QR para vincularlo a su cuenta.",
        infoText__unableToScan: "Configure un nuevo m\xE9todo de inicio de sesi\xF3n en su autenticador e ingrese la clave que se proporciona a continuaci\xF3n.",
        inputLabel__unableToScan1: "Aseg\xFArese de que las contrase\xF1as basadas en el tiempo o de un solo uso est\xE9n habilitadas, luego termine de vincular su cuenta.",
        inputLabel__unableToScan2: "Alternativamente, si su autenticador admite TOTP URIs, tambi\xE9n puede copiar el URI completo."
      },
      removeResource: {
        messageLine1: "El c\xF3digo de verificaci\xF3n de este autenticador ya no ser\xE1 necesario al iniciar sesi\xF3n.",
        messageLine2: "Es posible que su cuenta no sea tan segura. \xBFEst\xE1s seguro de que quieres continuar?",
        successMessage: "Se elimin\xF3 la verificaci\xF3n en dos pasos a trav\xE9s de la aplicaci\xF3n de autenticaci\xF3n.",
        title: "Eliminar la verificaci\xF3n en dos pasos"
      },
      successMessage: "La verificaci\xF3n en dos pasos ahora est\xE1 habilitada. Al iniciar sesi\xF3n, deber\xE1 ingresar un C\xF3digo de verificaci\xF3n de este autenticador como un paso adicional.",
      title: "Agregar aplicaci\xF3n de autenticaci\xF3n",
      verifySubtitle: "Ingrese el C\xF3digo de verificaci\xF3n generado por su autenticador",
      verifyTitle: "C\xF3digo de verificaci\xF3n"
    },
    mobileButton__menu: "Men\xFA",
    navbar: {
      account: "Profile",
      description: "Manage your account info.",
      security: "Security",
      title: "Account"
    },
    passkeyScreen: {
      removeResource: {
        messageLine1: void 0,
        title: void 0
      },
      subtitle__rename: void 0,
      title__rename: void 0
    },
    passwordPage: {
      checkboxInfoText__signOutOfOtherSessions: "It is recommended to sign out of all other devices which may have used your old password.",
      readonly: "Your password can currently not be edited because you can sign in only via the enterprise connection.",
      successMessage__set: "Su contrase\xF1a ha sido establecida.",
      successMessage__signOutOfOtherSessions: "Todos los dem\xE1s dispositivos han cerrado sesi\xF3n.",
      successMessage__update: "Tu contrase\xF1a ha sido actualizada.",
      title__set: "Configurar la clave",
      title__update: "Cambiar contrase\xF1a"
    },
    phoneNumberPage: {
      infoText: "Se enviar\xE1 un mensaje de texto con un enlace de verificaci\xF3n a este n\xFAmero de tel\xE9fono.",
      removeResource: {
        messageLine1: "{{identifier}} ser\xE1 eliminado de esta cuenta.",
        messageLine2: "Ya no podr\xE1 iniciar sesi\xF3n con este n\xFAmero de tel\xE9fono.",
        successMessage: "{{phoneNumber}} ha sido eliminado de su cuenta.",
        title: "Eliminar n\xFAmero de tel\xE9fono"
      },
      successMessage: "{{identifier}} ha sido a\xF1adido a tu cuenta.",
      title: "Agregar el n\xFAmero de tel\xE9fono",
      verifySubtitle: "Enter the verification code sent to {{identifier}}",
      verifyTitle: "Verify phone number"
    },
    profilePage: {
      fileDropAreaHint: "Cargue una imagen JPG, PNG, GIF o WEBP de menos de 10 MB",
      imageFormDestructiveActionSubtitle: "Eliminar la imagen",
      imageFormSubtitle: "Cargar imagen",
      imageFormTitle: "Imagen de perfil",
      readonly: "Your profile information has been provided by the enterprise connection and cannot be edited.",
      successMessage: "Tu perfil ha sido actualizado.",
      title: "Actualizar Cuenta"
    },
    start: {
      __experimental_passkeysSection: {
        menuAction__destructive: void 0,
        menuAction__rename: void 0,
        title: void 0
      },
      activeDevicesSection: {
        destructiveAction: "Cerrar sesi\xF3n en el dispositivo",
        title: "Dispositivos activos"
      },
      connectedAccountsSection: {
        actionLabel__connectionFailed: "Int\xE9ntelo nuevamente",
        actionLabel__reauthorize: "Authorize now",
        destructiveActionTitle: "Quitar",
        primaryButton: "Conectar cuenta",
        subtitle__reauthorize: "The required scopes have been updated, and you may be experiencing limited functionality. Please re-authorize this application to avoid any issues",
        title: "Cuentas conectadas"
      },
      dangerSection: {
        deleteAccountButton: "Eliminar cuenta",
        title: "Eliminar cuenta"
      },
      emailAddressesSection: {
        destructiveAction: "Eliminar direcci\xF3n de correo electr\xF3nico",
        detailsAction__nonPrimary: "Establecer como primario",
        detailsAction__primary: "Completar la verificaci\xF3n",
        detailsAction__unverified: "Completar la verificaci\xF3n",
        primaryButton: "Agregar una direcci\xF3n de correo electr\xF3nico",
        title: "Correos electr\xF3nicos"
      },
      enterpriseAccountsSection: {
        title: "Enterprise accounts"
      },
      headerTitle__account: "Cuenta",
      headerTitle__security: "Seguridad",
      mfaSection: {
        backupCodes: {
          actionLabel__regenerate: "Regenerar c\xF3digos",
          headerTitle: "C\xF3digos de respaldo",
          subtitle__regenerate: "Obtenga un nuevo conjunto de c\xF3digos de respaldo seguros. Los c\xF3digos de respaldo anteriores se eliminar\xE1n y no podr\xE1n ser usados.",
          title__regenerate: "Regenerar c\xF3digos de respaldo"
        },
        phoneCode: {
          actionLabel__setDefault: "Establecer por defecto",
          destructiveActionLabel: "Eliminar n\xFAmero telef\xF3nico"
        },
        primaryButton: "A\xF1adir verificaci\xF3n de dos pasos",
        title: "Verificaci\xF3n de dos pasos",
        totp: {
          destructiveActionTitle: "Eliminar",
          headerTitle: "Aplicaci\xF3n de autenticaci\xF3n"
        }
      },
      passkeysSection: {
        menuAction__destructive: void 0,
        menuAction__rename: void 0,
        title: void 0
      },
      passwordSection: {
        primaryButton__setPassword: "Establecer contrase\xF1a ",
        primaryButton__updatePassword: "Cambiar contrase\xF1a",
        title: "Contrase\xF1a"
      },
      phoneNumbersSection: {
        destructiveAction: "Quitar n\xFAmero de tel\xE9fono",
        detailsAction__nonPrimary: "Establecer como primario",
        detailsAction__primary: "Completar la verificaci\xF3n",
        detailsAction__unverified: "Completar la verificaci\xF3n",
        primaryButton: "Agregar un n\xFAmero de tel\xE9fono",
        title: "N\xFAmeros telef\xF3nicos"
      },
      profileSection: {
        primaryButton: "",
        title: "Perfil"
      },
      usernameSection: {
        primaryButton__setUsername: "Crear nombre de usuario",
        primaryButton__updateUsername: "Cambiar nombre de usuario",
        title: "Nombre de usuario"
      },
      web3WalletsSection: {
        destructiveAction: "Quitar cartera",
        primaryButton: "Web3 cartera",
        title: "Web3 cartera"
      }
    },
    usernamePage: {
      successMessage: "Su nombre de usuario ha sido actualizado.",
      title__set: "Actualizar nombre de usuario",
      title__update: "Actualizar nombre de usuario"
    },
    web3WalletPage: {
      removeResource: {
        messageLine1: "{{identifier}} ser\xE1 eliminado de esta cuenta.",
        messageLine2: "Ya no podr\xE1 iniciar sesi\xF3n con esta billetera web3.",
        successMessage: "{{web3Wallet}} ha sido eliminado de su cuenta.",
        title: "Eliminar la billetera web3"
      },
      subtitle__availableWallets: "Seleccione una billetera web3 para conectarse a su cuenta.",
      subtitle__unavailableWallets: "No hay billetera web3 disponibles.",
      successMessage: "La billetera ha sido agregada a su cuenta.",
      title: "A\xF1adir web3 billetera"
    }
  }
};
// Annotate the CommonJS export names for ESM import in node:
0 && (module.exports = {
  esES
});
